config/recipe.yml

```diff
@@ -15,8 +15,8 @@ modules:
     files:
       - usr: /usr # copy static configurations
                   # configuration you wish to end up in /etc/ on the booted system should be 
-                  # added into /usr/etc/ as that is the proper "distro" config directory on ostree
-                  # read more in the files module's README
+                  # added into /usr/etc/ (under /config/files) as that is the proper "distro" 
```

Comment:
This is false, it is added into /usr/etc/, there's no such thing as config/files/ on the OS. I agree that additional clarification would be useful, but this just makes things more confusing.

Reply:
ah whoops, when I read this comment I interpreted "added into" as "that you added into" as in me, the repo owner, added to the repo in that location. Let me change this